server/config.ts

```diff
@@ -52,6 +52,7 @@ export default {
     connectionString: get('APPLICATIONINSIGHTS_CONNECTION_STRING', null, requiredInProduction),
     cloudRoleName: 'interventions-ui',
     excludedRequests: [/^GET \/assets\/.*/, /^GET \/health.*/],
+    errorStatusCodes: [500],
```

Comment:
is it possible to have this over the entire range of 500..599? might be simpler to hardcode it as `envelope.data.baseData.responseCode >= 500`?

Reply:
good idea